test/aspect/execute_tests_impl.py

```diff
@@ -157,7 +158,8 @@ def verify_test(test: TestCase, process: sb.CompletedProcess, verbose: bool) ->
 
 
 def make_cmd(test_cmd: TestCmd, extra_args: List[str]) -> List[str]:
-    cmd = ["bazelisk", "build", "--noshow_progress"]
+    bazel = which("bazelisk") or which("bazel")
```

Comment:
It is common for teams to use `bazelisk` but rename it to `bazel`. That's what we have done on our end, so the intent is just to use whichever one is available but prefer `bazelisk` if both are available.